src/test.ts

```diff
@@ -0,0 +1,60 @@
+import { composeReducers, Result } from "./redux-reducer-effects";
+import { assert } from "chai";
+
+type Action = { type : string};
+type Task = { task: string };
+type State = {
+  counter: number,
+};
+
+describe("redux-reducer-effects", function() {
+
+  describe("compose reducers", function() {
+
+    it("keeps all state changes", function() {
+      const combined = composeReducers(increment, increment, increment);
+
+      const [state, _ts] = <any>(combined({ counter: 0 }, { type: "init" }));
```

Comment:
As in `State | [State, Task[]]`? Can we type `compose/combine` so they always return the latter, as they do, don't they?